eZ/Publish/Core/Repository/LocationService.php

```diff
@@ -458,6 +458,10 @@ public function updateLocation(APILocation $location, LocationUpdateStruct $loca
             throw new InvalidArgumentValue('priority', $locationUpdateStruct->priority, 'LocationUpdateStruct');
```

Comment:
Looked at it briefly, so please make sure, but it seems to me it's a good chance to move the above check to `isValidLocationPriority` also.